packages/web-components/fast-foundation/src/tooltip/tooltip.ts

```diff
@@ -395,6 +395,62 @@ export class Tooltip extends FoundationElement {
                 this.horizontalDefaultPosition = this.position;
                 break;
 
+            case TooltipPosition.topLeft:
+                this.verticalPositioningMode = "locktodefault";
```

Comment:
Seems like everything but the default case uses `locktodefault`. Can we factor that out and reduce duplication?